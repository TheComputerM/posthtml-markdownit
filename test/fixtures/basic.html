<md class="lol">
    # Markdown, Yeah!
    ---
    > Working !
</md>