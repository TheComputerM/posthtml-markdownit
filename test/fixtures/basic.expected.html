<div class="lol">
    <h1>Markdown, Yeah!</h1>
    <hr>
    <blockquote>
    <p>Working !</p>
    </blockquote>

</div>